articles/openshift/oshift-how-portworx-snapshots.md

```diff
@@ -0,0 +1,347 @@
+          storage: 2Gi" | oc create -f -
+    ```
+
+2. To create a cloud snapshot, create a `VolumeSnapshot` object with the `portworx/snapshot-type=cloud` annotation:
+
+    ```none
+    echo "apiVersion: volumesnapshot.external-storage.k8s.io/v1
+    kind: VolumeSnapshot
+    metadata:
+      name: test-cloudsnap
+      namespace: test-snapshot-1
+      annotations:
+        portworx/snapshot-type: cloud
+    spec:
+      persistentVolumeClaimName: test-cloudsnap" | oc create -f -
+    ```
+
+    You can also provide the cloud credentials for your chosen cloud storage in the `portworx/cloud-cred-id` annotation, however this is only necessary when there are multiple cloud storage destinations. In our Portworx clusters out of the box you won't need to specify this but it would look like the below:
+
+    ```none
+    apiVersion: volumesnapshot.external-storage.k8s.io/v1
+    kind: VolumeSnapshot
+    metadata:
+      name: test-cloudsnap
+      namespace: test-snapshot-1
+      annotations:
+        portworx/snapshot-type: cloud
+        portworx/cloud-cred-id: <cloud-cred-id>
+    spec:
+      persistentVolumeClaimName: test-cloudsnap
+    ```
+
+3. The restore process is exactly the same as the local on-demand snapshots: create a PVC in the `stork-snapshot-sc` storage class and give it the `snapshot.alpha.kubernetes.io/snapshot` annotation with the name of the taken snapshot:
+
+    ```none
+    echo "apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      name: cloudsnap-restore
+      annotations:
+        snapshot.alpha.kubernetes.io/snapshot: test-cloudsnap
+    spec:
+      accessModes:
+         - ReadWriteOnce
+      storageClassName: stork-snapshot-sc
+      resources:
+        requests:
+          storage: 2Gi" | oc create -f -
+    ```
+
+## Creating scheduled snapshots
+
+You can create snapshot schedules that take recurring backups at a certain time each day, week, month or at a specified interval. For more information on each of the options see <https://docs.portworx.com/portworx-install-with-kubernetes/storage-operations/create-snapshots/scheduled/>.
+
+The following example creates a daily policy that creates snapshots at a certain time each day:
+
+1. Create some `SchedulePolicy` objects:
+
+    ```none
+    echo "apiVersion: stork.libopenstorage.org/v1alpha1
+    kind: SchedulePolicy
+    metadata:
+      name: daily
+    policy:
+      daily:
+        time: '10:00PM'
+        retain: 3" | oc create -f -
+    ```
+
+    ```none
+    echo "apiVersion: stork.libopenstorage.org/v1alpha1
+    kind: SchedulePolicy
+    metadata:
+      name: ten-minute
+    policy:
+      interval:
+        intervalMinutes: 10
+        retain: 3" | oc create -f -
+    ```
+
+2. Create a storage class that references one or more schedule policies. Any PVCs created from the storage class will have all schedule policies applied, For example, to backup the PVC locally every ten minutes and at 10:00PM daily to cloud-storage:
```

Comment:
Here you don't need to describe the policies now, just refer to them as described above.